roles/sysadm/templates/root_bashrc.j2

```diff
@@ -0,0 +1,143 @@
+# ~/.bashrc: executed by bash(1) for non-login shells.
+# see /usr/share/doc/bash/examples/startup-files (in the package bash-doc)
+# for examples
+
+# If not running interactively, don't do anything
+[ -z "$PS1" ] && return
+
+export LC_ALL="en_US.UTF-8"
+export LANG="en_US"
+
+function parse_git_dirty {
+  [[ $(git status 2> /dev/null | tail -n1) != "nothing to commit, working tree clean" ]] && echo "*"
+}
+
+function env_set {
+  if [ -z "$ENV" ]; then
+    echo ""
+    else
+    echo "($ENV)"
+  fi
+}
+
+function parse_git_branch {
+  git branch --no-color 2> /dev/null | sed -e '/^[^*]/d' -e "s/* \(.*\)/[\1$(parse_git_dirty)]/"
+}
+
+RED="\[\e[0;31m\]"
+GRAY="\[\e[0;37m\]"
+YELLOW="\[\e[0;33m\]"
+BLUE="\[\e[0;34m\]"
+GREEN="\[\e[0;32m\]"
+WHITE="\[\e[1;37m\]"
+LIGHT_GREEN="\[\e[1;32m\]"
+TXTRST='\[\e[0m\]'
+
+export CLICOLOR=1
+# export LSCOLORS=ExFxCxDxBxegedabagacad
+export PS1="[$RED\t \u@\h:$YELLOW\w$TXTRST]$RED\$(parse_git_branch)$TXTRST\$(env_set) # \[\e[m\]"
```

Comment:
Please rm the git prompt additions. It's handy, I love it, but from fixing broken boxes I've learned painfully not to add a moving part to root's command prompt.